Build failure: wbg-1.2.0

## Steps To Reproduce

Steps to reproduce the behavior:

1. build *wbg*

## Build log

<!-- insert build log in code block in collapsable section -->

<details>

<summary>Build Log</summary>

```
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/a5v2gww1iixx8fqswzznc02l5v7m6608-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
mesonConfigurePhase flags: --prefix=/nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0 --libdir=/nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/lib --libexecdir=/nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/libexec --bindir=/nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/bin --sbindir=/nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/sbin --includedir=/nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/include --mandir=/nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/share/man --infodir=/nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/share/info --localedir=/nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/share/locale -Dauto_features=enabled -Dwrap_mode=nodownload --buildtype=release -Dpng=enabled -Djpeg=enabled -Dwebp=enabled
[1mThe Meson build system[0m
Version: 1.6.0
Source dir: [1m/build/source[0m
Build dir: [1m/build/source/build[0m
Build type: [1mnative build[0m
Project name: [1mwbg[0m
Project version: [1m1.2.0[0m
C compiler for the host machine: [1mgcc[0m (gcc 14.2.1 "gcc (GCC) 14.2.1 20241116")
C linker for the host machine: [1mgcc[0m ld.bfd 2.43.1
Host machine cpu family: [1mx86_64[0m
Host machine cpu: [1mx86_64[0m
Compiler for C supports arguments -fmacro-prefix-map=/foo=: [1;32mYES[0m 
Library [1mm[0m found: [1;32mYES[0m
Found pkg-config: [1;32mYES[0m [1m(/nix/store/0d4m43yp69lrm8imxbqgl9zxjwwz52jw-pkg-config-wrapper-0.29.2/bin/pkg-config)[0m [1;34m0.29.2[0m
Run-time dependency [1mpixman-1[0m found: [1;32mYES[0m [36m0.44.2[0m
Run-time dependency [1mlibpng[0m found: [1;32mYES[0m [36m1.6.43[0m
Run-time dependency [1mlibjpeg[0m found: [1;32mYES[0m [36m3.0.4[0m
Run-time dependency [1mlibwebp[0m found: [1;32mYES[0m [36m1.4.0[0m
Run-time dependency [1mwayland-protocols[0m found: [1;32mYES[0m [36m1.38[0m
Run-time dependency [1mwayland-client[0m found: [1;32mYES[0m [36m1.23.1[0m
Run-time dependency [1mtllist[0m found: [1;32mYES[0m [36m1.1.0[0m
Build-time dependency [1mwayland-scanner[0m found: [1;32mYES[0m [36m1.23.1[0m
Program [1m/nix/store/qqzq7ci4iqlaiicfph4hhh58vfb0i9ck-wayland-scanner-1.23.1-bin/bin/wayland-scanner[0m found: [1;32mYES[0m (/nix/store/qqzq7ci4iqlaiicfph4hhh58vfb0i9ck-wayland-scanner-1.23.1-bin/bin/wayland-scanner)
Program [1menv[0m found: [1;32mYES[0m (/nix/store/4s9rah4cwaxflicsk5cndnknqlk9n4p3-coreutils-9.5/bin/env)
Build targets in project: [1m6[0m

wbg [36m1.2.0[0m

    PNG support  : [1;32mYES[0m
    JPEG support : [1;32mYES[0m
    WebP support : [1;32mYES[0m
    SVG support  : [1;32mYES[0m

  [1mUser defined options[0m
    auto_features: enabled
    bindir       : /nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/bin
    buildtype    : release
    includedir   : /nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/include
    infodir      : /nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/share/info
    jpeg         : enabled
    libdir       : /nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/lib
    libexecdir   : /nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/libexec
    localedir    : /nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/share/locale
    mandir       : /nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/share/man
    png          : enabled
    prefix       : /nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0
    sbindir      : /nix/store/7jkdimc9c8m0flhklcfzskpgnyk5rf4g-wbg-1.2.0/sbin
    webp         : enabled
    wrap_mode    : nodownload

Found ninja-1.12.1 at /nix/store/byvgvqlh007flwk940c99zfgzrjg0zsg-ninja-1.12.1/bin/ninja
mesonConfigurePhase: enabled\ parallel\ building
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j16
[1/17] Generating generate_version with a custom command
[2/17] Generating external_wlr_layer_shell_unstable_v1_xml-private-code with a custom command
[3/17] Generating external_wlr_layer_shell_unstable_v1_xml-client-header with a custom command
[4/17] Generating __nix_store_6qkg40r0qb6z0n9rhkyzlibclpi5wbfj_wayland_protocols_1_38_share_wayland_protocols_stable_xdg_shell_xdg_shell_xml-private-code with a custom command
[5/17] Generating __nix_store_6qkg40r0qb6z0n9rhkyzlibclpi5wbfj_wayland_protocols_1_38_share_wayland_protocols_stable_xdg_shell_xdg_shell_xml-client-header with a custom command
[6/17] Compiling C object wbg.p/meson-generated_.._xdg-shell.c.o
[7/17] Compiling C object wbg.p/meson-generated_.._wlr-layer-shell-unstable-v1.c.o
[8/17] Compiling C object wbg.p/webp.c.o
[9/17] Compiling C object wbg.p/jpg.c.o
[10/17] Compiling C object wbg.p/svg.c.o
[11/17] Compiling C object wbg.p/png.c.o
[12/17] Compiling C object wbg.p/log.c.o
[13/17] Compiling C object wbg.p/shm.c.o
[14/17] Compiling C object wbg.p/main.c.o
[15/17] Compiling C object wbg.p/nanosvgrast.c.o
[16/17] Compiling C object wbg.p/nanosvg.c.o
FAILED: wbg.p/nanosvg.c.o 
gcc -Iwbg.p -I. -I.. -I../3rd-party/nanosvg/src -I/nix/store/5z7yxa99nghyvjrqspzlimgfsydc2an7-pixman-0.44.2/include/pixman-1 -I/nix/store/j0cx534qlwqdinajjm4f7l3c2460785l-libpng-apng-1.6.43-dev/include/libpng16 -I/nix/store/6sxnhzr0i1r6bysqm4p9smvq4gjj2s60-libjpeg-turbo-3.0.4-dev/include -I/nix/store/75v06fahb45kf0rcq37957n93jxka2cf-libwebp-1.4.0/include -I/nix/store/yan7xnjlp3l7pg82qz46b7z7x9za4flf-wayland-1.23.1-dev/include -I/nix/store/k584jqg0srmgp4fhfxgl6izkxjch92sq-tllist-1.1.0/include -fdiagnostics-color=always -DNDEBUG -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Werror -std=c18 -O3 -D_POSIX_C_SOURCE=200809L -D_GNU_SOURCE -fmacro-prefix-map=../= -DWBG_HAVE_PNG=1 -DWBG_HAVE_JPG=1 -DWBG_HAVE_WEBP=1 -DWBG_HAVE_SVG=1 -MD -MQ wbg.p/nanosvg.c.o -MF wbg.p/nanosvg.c.o.d -o wbg.p/nanosvg.c.o -c ../nanosvg.c
In file included from ../nanosvg.c:6:
../3rd-party/nanosvg/src/nanosvg.h: In function 'nsvg__parseTransform':
../3rd-party/nanosvg/src/nanosvg.h:1656:15: error: 't[0]' may be used uninitialized [-Werror=maybe-uninitialized]
 1656 |         float t[6];
      |               ^
../3rd-party/nanosvg/src/nanosvg.h:1656:15: error: 't[1]' may be used uninitialized [-Werror=maybe-uninitialized]
../3rd-party/nanosvg/src/nanosvg.h:1656:15: error: 't[2]' may be used uninitialized [-Werror=maybe-uninitialized]
../3rd-party/nanosvg/src/nanosvg.h:1656:15: error: 't[3]' may be used uninitialized [-Werror=maybe-uninitialized]
../3rd-party/nanosvg/src/nanosvg.h:1656:15: error: 't[4]' may be used uninitialized [-Werror=maybe-uninitialized]
../3rd-party/nanosvg/src/nanosvg.h:1656:15: error: 't[5]' may be used uninitialized [-Werror=maybe-uninitialized]
cc1: all warnings being treated as errors
ninja: build stopped: subcommand failed.
```

</details>

## Additional context

<!-- Add any other context about the problem here. -->

## Metadata

<!-- Please insert the output of running `nix-shell -p nix-info --run "nix-info -m"` below this line -->
```
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.66, NixOS, 25.05 (Warbler), 25.05.20241219.d70bd19`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.91.1
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux, x86_64-v4-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/mat/.config/nix/nix.conf:/nix/store/sva7lp0ax2816racaphz5zw5jylkwc51-stylix-kde-config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/mat/.local/share/flatpak/exports/etc/xdg/nix/nix.conf:/var/lib/flatpak/exports/etc/xdg/nix/nix.conf:/home/mat/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/mat/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/mat/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/873hdnqclw6c5kqax9mmw57a72rn4c3h-lix-2.91.1/share`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/8fwsiv0hd7nw1brkvka0jf1frk3m7qkr-source`
```

## Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->
@AndersonTorres 
---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/wb/wbg/package.nix
+++ b/pkgs/by-name/wb/wbg/package.nix
@@ -57,6 +57,10 @@ stdenv.mkDerivation rec {
     (lib.mesonEnable "webp" enableWebp)
   ];
 
+  env.NIX_CFLAGS_COMPILE = toString [
+    "-Wno-error=maybe-uninitialized"
+  ];
+
   meta = with lib; {
     description = "Wallpaper application for Wayland compositors";
     homepage = "https://codeberg.org/dnkl/wbg";

